turbinia/api/openapi.yaml

```diff
@@ -491,17 +521,91 @@ paths:
       summary: Read Jobs
       tags:
       - Turbinia Jobs
-  /api/logs/{query}:
+  /api/logs/api_server:
+    get:
+      description: Retrieve log data.
+      operationId: get_api_server_logs
+      parameters:
+      - in: query
+        name: num_lines
+        required: false
+        schema:
+          default: 500
+          exclusiveMinimum: 0.0
+          title: Num Lines
+          type: integer
+      responses:
+        '200':
+          content:
+            application/json:
```

Comment:
is this an json response? or plain text like the config.download endpoint.
same for the other log endpoints.